Mathlib/Tactic/Linter/UnusedAssumptionInType.lean

```diff
@@ -0,0 +1,100 @@
+/-
+Copyright (c) 2024 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
```

Comment:
@grunweg Do you want to add yourself here?

Reply:
Thanks, that sounds reasonable.
```suggestion
Authors: Yury Kudryashov, Michael Rothgang